src/App.tsx

```diff
@@ -94,6 +94,10 @@ export const App = async (props: AppProps, container?: HTMLElement) => {
           props.theme.mainColor = config.mainColor;
         }
 
+        if(config.hideSignature) {
+          props.theme.showSignature = !config.hideSignature
+        }
```

Comment:
This approach probably allows setting `theme: {showSignature: false}` using `BotChat.App` settings params which is not OK. This prop should not be configurable from outside. Could be this proposed approach with setting value always possible solution?
```suggestion
        props.theme.showSignature = !config.hideSignature
```

Reply:
Yes, I think this should work. Will use this solution.